boards/b-l072z-lrwan1/Makefile.features

```diff
@@ -2,6 +2,7 @@ CPU = stm32
 CPU_MODEL = stm32l072cz
 
 # Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_adc
```

Comment:
`boards/b-l072z-lrwan1/Kconfig` will also need an update

Reply:
I made an additional push request. Sorry for missing this out.